lib/active_delivery/lines/base.rb

```diff
@@ -7,17 +7,21 @@ class Base
       attr_accessor :owner
       attr_writer :handler_class
 
+      DEFAULT_RESOLVER = ->(name) { name.gsub(/Delivery$/, "Notifier").safe_constantize }
```

Comment:
Do we need this constant? I can't see it used anywhere.
Also, we cannot use `#safe_constantize`, 'cause Base line could be used in a Rails-less context.